src/Transport/Connectivity/NamespaceManagerAdapter.cs

```diff
@@ -23,10 +23,14 @@ public NamespaceManagerAdapter(NamespaceManager manager)
         {
             try
             {
-                await manager.GetQueuesAsync().ConfigureAwait(false);
+                await manager.GetQueuesAsync("startswith(path, '$#_') eq true").ConfigureAwait(false);
 
                 return true;
             }
+            catch (MessagingEntityAlreadyExistsException)
```

Comment:
right, it's not needed anymore, I played with CreateQueue for a while, but reverted to this pattern